gridspeccer/cli.py

```diff
@@ -22,6 +22,11 @@ def plot():
         prog="gridspeccer",
         description="Plotting tool for easier poisitioning.",
     )
+    parser.add_argument(
+        "--ft",
```

Comment:
Should be `--filetype` imho, maybe with `--ft` as alias. But I would prefer expressivity in the API.

Reply:
fixed by 15d0f7f